packages/shared/src/components/tabs/TabList.tsx

```diff
@@ -30,17 +55,20 @@ function TabList({
               return;
             }
 
+            currentActiveTab.current = el;
             const rect = el.getBoundingClientRect();
 
             if (rect.width === 0) {
               return;
             }
 
-            const size = rect.width / 2;
-            const value = size + el.offsetLeft;
-            setOffset(value);
+            const size = rect.width;
+            const leftOffset = el.offsetLeft;
+            setOffset(leftOffset);
+            setIndicatorOffset(size / 2 + leftOffset);
```

Comment:
Could `indicatorOffset` just always be computed as this? To avoid one `useState`?

Reply:
Removed the states completely since we already have a hold of which element is currently active. https://github.com/dailydotdev/apps/pull/2928/files#diff-a6e56221ccd7c5f4fe9c333d7082e8439f4b34772c3555880f158ffba9fd66dd